### DISCUSSION TITLE
Layouts for custom local theme

### DISCUSSION THREAD

**Discussion by NiewView** [vuepress/core#135] [Q&A] [answered]
Hi Vuepress Community,
I'm currently trying to build a simple website with Vuepress and a custom local theme.
Currently it seems like my theme gets recognized, but layouts won't get registered.

In the theme I define each template individual (currently only the necessary ones):
```js
const fooTheme = {
  name: "vuepress-theme-foo",
  layouts: {
    Layout: path.resolve(__dirname, "./layouts/Layout.vue"),
    404: path.resolve(__dirname, "./layouts/404.vue"),
  },
};
```

But after running it in dev mode the output in `layoutComponents.js` stays empty:

```js
import { defineAsyncComponent } from 'vue'

export const layoutComponents = {
}
```

I uploaded a simple version of the problem into [this repo](https://github.com/NiewView/DemoVuepress).

Can someone pinpoint me to the mistake I am making or is it possibly a bug?

**Comment by meteorlxy** [comment, marked as answer]
😓 It's a typo in docs...

`module.export` -> `module.exports`

**Reply by NiewView** [reply]
🤦‍♂️🤦‍♂️🤦‍♂️
Thank you so much. I spent so many hours changing almost everything, but haven't seen the typo...

**Reply by meteorlxy** [reply]
🤦‍♂️Sorry. My fault. Have fixed it.
